lucien-ios/APIClient/LucienRequest.swift

```diff
@@ -10,27 +10,37 @@ import Foundation
 enum LucienRequest: Request {
 
     static let baseURL = "https://lucien-server-staging.herokuapp.com"
-    static let acceptHeader = "application/vnd.lucien-app.com; version=1"
-    static var authorizationHeader: String?
+    static let acceptHeader: String? = "application/vnd.lucien-app.com; version=1"
```

Comment:
The acceptHeader is supposed to remain the same throughout all the requests -- should I keep it this way?

Reply:
If it stays the same, and it's not nil a) it doesn't need to be optional and b) you don't need the type declaration (`: String?`) because the compiler can infer that `acceptHeader` is a string from the definition.